### ISSUE TITLE
[Picker][TagPicker][Accessibilty] Search suggestions of tags are not announced by Narrator

### ISSUE THREAD

**Issue by aadhars** [OfficeDev/office-ui-fabric-react#3715]
### Bug Report

- __Package version(s)__: Latest
- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) No

Requested priority: (Blocking, High, Normal, Low) Normal

Products/sites affected: (if applicable) VSTS

#### Describe the issue:

Narrator is not announcing the suggestion that appear in tag picker control.User is not informed of about availability of suggestions and the number of suggestions.

#### Actual behavior:

Narrator is not announcing the search suggestions of tags.if we use arrow keys focus moves to the next suggestions.But user is not informed of about availability of suggestions and the number of suggestions.

#### Expected behavior:
Narrator should announce the search suggestions once the suggestion are populated and it should announce the number of results.

### If applicable, please provide a codepen repro:
It can be observed on fabric site

https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/pickers

**Comment by michaelangotti** [comment]
@betrue-final-final Do you know if this is something we want to support? We aren't sure if this would be expected behavior for this kind of navigation.

**Comment by betrue-final-final** [comment]
Yes, it should do this. We'll want to check if there's a formal accessibility spec, and if not, make one so this is done right.

Rule of thumb for accessibility - Equity: If a sighted person gets visual information that helps them accomplish their task, then a non-sighted person needs an equivalent piece of information.

**Comment by natalieethell** [comment]
This might be something that we can solve in the new "Announced"/"Announcement" component we'll be working on, which is being tracked here: #5615.

**Comment by daweffe** [comment]
This bug is affecting Planner Accessibility compliance. Is it being worked on?

**Comment by dzearing** [comment]
@natalieethell @aneeshack4 @joschect Can you guys look at this and come up with some sort of clear expectation for partners? 

Also this might be a dupe. I'd like to make sure we kill all picker accessibility concerns. Jon I would assume SharePoint is either doing something specific or has a critical accessibility issue here. Aneesha can help if you need it, but we need this addressed.

**Comment by dzearing** [comment]
Just pinged @joschect offline who's checking up.

**Comment by joschect** [comment]
This has been fixed. It now announces when it's loading, when new suggestions are available and when one is selected it announces the amount in the list. For instance, it will say "Blue selected, 1 of 3"
